CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/database/institution/mongo/AssignmentManager.java

```diff
@@ -157,8 +157,8 @@ public static String mongoInsertAssignment(final Authenticator authenticator, fi
     @SuppressWarnings({ "PMD.CyclomaticComplexity", "PMD.ModifiedCyclomaticComplexity", "PMD.StdCyclomaticComplexity" })
     public static SrlAssignment mongoGetAssignment(final Authenticator authenticator, final DB dbs, final String authId, final String assignmentId,
             final long checkTime) throws AuthenticationException, DatabaseAccessException {
-        final DBRef myDbRef = new DBRef(dbs, ASSIGNMENT_COLLECTION, convertStringToObjectId(assignmentId));
-        final DBObject cursor = myDbRef.fetch();
+        final DBCollection collection = dbs.getCollection(getCollectionFromType(School.ItemType.ASSIGNMENT));
```

Comment:
Change  this to assignmentCollection or assignments please cause collection also means:
`database.getCollection(USER_GROUP_COLLECTION);`